lib/msal-electron-proof-of-concept/samples/msal-electron-sample-app/src/Graph.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as rp from 'request-promise';
```

Comment:
I personally like doing `import request from 'request-promise';`

Reply:
Did you mean `import * as request from 'request-promise'`? I've changed it to that, thanks!

Reply:
`import * as request` and `import request` should be functionality equivalent here.

Reply:
For some reason the latter didn't work, any idea of why that could be?

Reply:
Oh strange. Not sure, don't worry about it in that case.